pootle/apps/virtualfolder/utils.py

```diff
@@ -0,0 +1,220 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) Pootle contributors.
+#
+# This file is a part of the Pootle project. It is distributed under the GPL3
+# or later license. See the LICENSE file for a copy of the license and the
+# AUTHORS file for copyright and authorship information.
+
+from fnmatch import fnmatch
+
+from pootle.core.url_helpers import split_pootle_path
+from pootle_fs.utils import PathFilter
+from pootle_store.models import Store
+
+from .models import VirtualFolder, VirtualFolderTreeItem
+
+
+def update_vfolder_tree(vf, store):
+    """For a given VirtualFolder and Store update the VirtualFolderTreeItem
+    """
+    # Create missing VirtualFolderTreeItem tree structure for affected Stores
+    # after adding or unobsoleting Units.
```

Comment:
this is just copied from the original https://github.com/translate/pootle/pull/5233/files#diff-362cf55e6d6252afb9292eeae52046d5L21

and again will be removed very soon